### ISSUE TITLE
[Feature Request] CreateAdditionalSwapChain support? (FFXIV Discussion)

### ISSUE THREAD

**Issue by Trucido** [PeterTh/gedosato#203]
currently Final Fantasy XIV doesn't work because it throws out an error saying this ins't supported. Would love to see this supported. I'd submit patches if I was a more skilled coder, and i have very little understanding how GeDaSaTo even works.

**Comment by EmpiresBane** [comment]
I am unsure when it changed, but I just tested it today and no longer get that error. Maybe it was a change in GeDoSaTo, or maybe it was something they changed in 2.5. Either way, I'm going to be trying to find a PSHash soon, because I would absolutely love to see some better AA. Not today or tomorrow, but sometime this week.

**Comment by Trucido** [comment]
Keep us posted if you find a working PSHash! I gave up. There was a few that were close but no cigar, but most just didn't work at all. There's just too many deferred rendering layers and I don't think GeDoSaTo was even dumping everything properly at the time. Also, speaking of... I noticed they tweaked the FXAA a bit in 2.5

Some hashes I tried:
injectPSHash 9e5dc2d9
injectPSHash 13c2e0fb
injectPSHash 3b3535e7
injectVSHash 642b73b6

**Comment by EmpiresBane** [comment]
I spent about 6 hours testing out a bunch of different hashes, and I could not get it working. 13c2e0fb seemed to be the best one, since when you used toggleHUD, it would actually disable the HUD. Unfortunately, none of the plugins had any effects. The odd thing is it's called twice. That might be related to the problem. There was also a hash that I tried that appears after drawing the HUD, and it managed to affect the game without affecting most of the HUD, but it also broke all text. I could find it again if you want to take a look at it. The annoying thing is that it's super obvious where they start working on the menu, but any changes before that don't seem to do anything.

**Comment by Trucido** [comment]
yeah I had exact same thing basically, it was either all or nothing. most
didn't work

**Comment by Drakonas** [comment]
It seems a recent update to FFXIV has broken GeDoSaTo support. Just a few days ago, it was loading fine (although I accidentally shut down XIV with the higher resolution on, and started GeDoSaTo afterwards, but it would boot fine after I started the game w/o GeDoSaTo and changed it back). And now it crashes every time GeDoSaTo is loaded (right after clicking the button on the launcher), whether I change it back to native 1080p@144Hz or not. My guess is something changed. Are any of you having issues?

Also, before this GeDoSaTo mess, I don't remember there being a big black window loading before going to fullscreen. That shows up every time now. Perhaps my memory is serving me incorrectly, as I used to run in Borderless Windowed mode before GeDoSaTo interested me.

**Comment by Trucido** [comment]
@Drakonas the only way I was able to get it working back in ARR was starting up the game in window mode or borderless window, and then changing to fullscreen after the game loads.

**Comment by Drakonas** [comment]
@Trucido It crashes in window mode for me too, if GeDoSaTo is enabled.

**Comment by Trucido** [comment]
@Drakonas if you think it might be an issue of starting up in a higher res, you could try editing the .ini file and change the res. it's in Documents\My Games\FFXIV-blahblah\something\

**Comment by Drakonas** [comment]
@Trucido It's already set to my native 1080p resolution. Like I said, I had it running, but I think a recent update broke it... or something else happened.

**Comment by Drakonas** [comment]
@Trucido Yeah, the FFXIV boot executable (launcher, separate from the game) was updated near the end of June this year. That might be what's causing my problem. I'm seeing if anyone else has the same issues on Reddit.
